bigbluebutton-html5/imports/ui/components/whiteboard/whiteboard-overlay/cursor-listener/component.jsx

```diff
@@ -36,6 +36,7 @@ export default class CursorListener extends Component {
     this.handleTouchMove = this.handleTouchMove.bind(this);
     this.handleTouchEnd = this.handleTouchEnd.bind(this);
     this.handleTouchCancel = this.handleTouchCancel.bind(this);
+    this.clearTouchEvents = this.clearTouchEvents.bind(this);
```

Comment:
There's no reason to bind this function because it isn't sent anywhere and isn't an event listener.